mlir/utils/jenkins/Dockerfile

```diff
@@ -68,22 +68,16 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
   libnuma-dev \
   software-properties-common
 
-RUN apt-add-repository ppa:deadsnakes/ppa && apt-get update \
-     && apt-get install -y --no-install-recommends \
-            python3.8 python3.8-dev python3.8-venv python3-psutil \
-            python3-pip python3-setuptools \
-     && update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.8 10 \
-     && apt upgrade \
-     && python3 -m pip install --upgrade pip \
-     && pip3 install pandas numpy scipy jinja2 tomli \
-     && pip3 install -r requirements.txt \
-     && sed -i -e 's#/usr/bin/python3#/usr/bin/python3.6#' /usr/bin/apt-add-repository /usr/bin/add-apt-repository
+RUN apt-get update && apt-get install -y --no-install-recommends \
+           python3.8 python3.8-dev python3.8-venv python3-psutil \
+           python3-pip python3-setuptools \
+    && update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.8 10 \
+    && apt upgrade \
+    && python3 -m pip install --upgrade pip \
+    && pip3 install pandas numpy scipy jinja2 tomli \
```

Comment:
When I tried to build another image on top of this one, it had the error `apt-add-repository: not found`. 
And it worked when I removed this chunk and only kept this line.
Can you give it a try and confirm?

Reply:
???

Maybe the `update-alternatives` line is the problem - I'd poke at this, but if the base Docker image works, you can paper over problems in the sub-images.

Reply:
Wait, hold on, `apt-add-repositories` should be using system python, which, on 20.04 is 3.8?

Reply:
```
% /usr/bin/python3 --version
Python 3.8.10
```

Reply:
Ah. It should be the `sed` line that replaced python3 with python3.6.

Reply:
That sed line is removed now - it was only there because in Ubuntu 18.04 we were manually upgrading to Python 3.8 but needed to keep the apt tools pointed at python3.6